src/azul/chalice.py

```diff
@@ -187,6 +187,13 @@ def _register_handler(self,
             assert wrapped_handler.lambda_name == name
         else:
             wrapped_handler.lambda_name = name
+        # We store any other parameters used to configure the handler for the
+        # same reason.
+        for k, v in kwargs.items():
+            if hasattr(wrapped_handler, k):
```

Comment:
Some duplication here that could be eliminated by appending a tuple to what .items() returns or by extracting a nested function.

Also, I would like to understand when we expect this condition to ever be True.

Reply:
This condition is true when a handler has multiple decorators, for example https://github.com/DataBiosphere/azul/blob/65970a0947f38fae439a3bf8fd960d351787b7a3/lambdas/service/app.py#L1184

Reply:
One weird side-effect of this is that the `path` attribute of handlers is being overwritten. The path attribute was added a long time ago and is used only by `file_url()` as far as I can tell. I had to add some logic to work around this.

Reply:
Not seeing any of the stuff we talked about.

Reply:
I have since addressed this.